telethon/network/mtprotosender.py

```diff
@@ -357,14 +358,15 @@ async def _reconnect(self, last_error):
         self._state.reset()
 
         retries = self._retries if self._auto_reconnect else 0
-        for attempt in retry_range(retries):
+        
+        attempt = 0
+        for attempt in retry_range(retries, False):
```

Comment:
```suggestion
        # We're already "retrying" to connect, so we don't want to force retries
        for attempt in retry_range(retries, force_retry=False):
```

Reply:
I used named args but was told to depricate it.